docs/user_manual/getting_verified.md

```diff
@@ -32,23 +32,21 @@ because that server is where most game servers are listed.
 Getting registered for the login server run by Griffon26 is an automated process:
 
 1. First log in to the Hirez servers
+1. Join a game server
+1. Open the Chat Console. (This might not have a keybinding, add one through `Settings > Keybindings > Interface`). Note: This console is different to the Unreal Console (~)
+1. Send a private chat message to `@taserverbot` with only the word `authcode` in it:
 
-2. Send a private chat message to `taserverbot` with only the word authcode in it:
+    `@taserverbot authcode`
 
```

Comment:
Technically you are sending a private message to taserverbot. The @ is how you send a private message, not who you send it to.
I assume that the "Say>" part caused confusion on whether you should type that yourself or not?
I think we should fix this all by using the technically accurate description that I had before, but then adding an animated screenshot of what it looks like to type this.

Reply:
Yeah, that was confusing. Hope the update covers that in more detail.